.github/workflows/dsa.yml

```diff
@@ -17,14 +17,15 @@ env:
 
 jobs:
   build:
+    if: false # both thumbv7 and wasm require a getrandom backend to be injected
```

Comment:
Why is `getrandom` a hard dependency now?

Reply:
wait? is it?

Reply:
Offhand I don't see anything which would make `getrandom` a hard dependency so I guess I'm confused why this is needed

Reply:
yeah you're correct, I guess I got confused with the other rand_core upgrade? I don't remember.